cgi/Project.pm

```diff
@@ -93,6 +95,16 @@ sub EvaluateCandidacy
 }
 
 # ========== REVIEW INTERFACE ========== #
+# Called by CRMS::LoadQueueForProject to prioritize candidates for the queue.
+# Return undef for no additional order (the default), or
+# a column name in bibdata (b.*) or candidates (c.*).
+# Example: 'b.author DESC'
+sub queue_order {
```

Comment:
If the body of this sub is essentially a no-op, I'd try just `sub queue_order {}`. I _think_ that's equivalent.

Reply:
Sure is, works for me.